services/functional-tests/src/test/resources/arquillian.xml

```diff
@@ -20,6 +20,7 @@
         <property name="env.setup.script.url">setup.sh</property>
         <property name="env.teardown.script.url">destroy.sh</property>
         <property name="namespace.use.current">true</property>
+        <property name="namespace.destroy.timeout">30000</property>
```

Comment:
30000 seconds?

Reply:
Whoops! I wasn't sure about the timeunit haha